### ISSUE TITLE
Question concerning usage

### ISSUE THREAD

**Issue by chaseboren** [mrplants/py2tex#1]
What is the separate use case from https://github.com/gpoore/pythontex ?

**Comment by mrplants** [comment]
Good question! It's kinda nuanced, but that add-on requires multiple passes of the pdflatex compiler, since their original pythontex doc is still strictly latex. My utility starts with a strictly non-latex format. This means it requires a single pass of the conversion utility, followed by however many passes pdflatex would normally need.

This makes it easier (for me at least) to combine the py2tex utility with latexmk (which was the overall goal).

Basically, it's just a little less complex.

**Comment by chaseboren** [comment]
Cool, thanks for that. I am playing around with something similar, though I would be using python as the runner for latex.  So you would have a really light markup language, processed by python to latex, and then python orchestrates running latex.

**Comment by mrplants** [comment]
You're making your own latex compiler in Python?  That's awesome!
